.changeset/pink-yaks-jam.md

```diff
@@ -0,0 +1,5 @@
+---
+'vite-imagetools': minor
```

Comment:
@JonasKruckenberg do you have any opinion about whether this fix should be `patch` or `minor`?

Reply:
I wasn't sure, either. It's technically a bug fix but it also technically adds functionality.

Reply:
It doesn’t really matter, but I feel `patch` is most appropriate bc it’s about fixing compatibility with frameworks right?